contracts/ValidatorSmartContractInterface.sol

```diff
@@ -0,0 +1,9 @@
+// Interface for contracts used to select validators 
+  
+pragma solidity >=0.5.0;
+  
+interface ValidatorSmartContractInterface {
+
+    function getValidators() external view returns (address[] memory);
+
```

Comment:
It might make sense to define a `ValidatorSetChanged` event here to be implemented by the calling contract.